### DISCUSSION TITLE
Bpm range in lazer is too small

### DISCUSSION THREAD

**Discussion by Morilli** [ppy/osu#15016] [Ideas]
The supported bpm range in lazer seems to be `(1 - 10,000)`, which is not enough for a lot of special use cases, like mania maps which use SVs.

Examples of maps that look visually wrong when their used bpm is not supported:
- https://osu.ppy.sh/beatmapsets/884981#mania/1849726 (at around 3:14)
- https://osu.ppy.sh/beatmapsets/1182702#mania/2465806 (the entire SV parts of the map, basically)

**Comment by bdach** [comment]
https://github.com/ppy/osu/discussions/14835
https://github.com/ppy/osu/discussions/13158

**Reply by Morilli** [reply]
While velocity is definitely an issue as well, none of these mentioned issues being fixed will affect the bpm range, which still matters.

To further give examples:
- Some bpm values are not parsed at all, with an exception thrown that reads `Value is too low` or `Value is too high`, which sometimes causes hit objects in standard mode to not appear at all (iirc), and definitely break things even if not because in stable those values are definitely read.
- bpm values matter in mania SV maps, even if velocity were to be unbounded: the bpm of a section determines how "high" on the screen a note will appear stuck (see first example map I gave above; it's relatively hard to explain. Basically in a time window of 1ms a single note will travel from being somewhere on-screen to the bottom).
